src/pages/Home/index.js

```diff
@@ -0,0 +1,534 @@
+                    </Typography>
+                    <Typography align="center">
+                      Master English with easy to understand courses
+                    </Typography>
+                  </CardContent>
+                </Card>
+              </Grid>
+              <Grid item xs={12} ms={6} md={4}>
+                <Card className={classes.card}>
+                  <CardMedia
+                    component="img"
+                    image={require("./assets/soft-skills.svg")}
+                    alt="green iguana"
+                  />
+                  <CardContent className={classes.cardConten}>
+                    <Typography
+                      gutterBottom
+                      variant="h6"
+                      align="center"
+                      component="div"
+                    >
+                      Soft Skills
+                    </Typography>
+                    <Typography align="center">
+                      Interview preparation to get you job ready
+                    </Typography>
+                  </CardContent>
+                </Card>
+              </Grid>
+              <Grid item xs={12} ms={6} md={4}>
+                <Card className={classes.card}>
+                  <CardMedia
+                    component="img"
+                    image={require("./assets/misc.svg")}
+                    alt="green iguana"
+                  />
+                  <CardContent className={classes.cardConten}>
+                    <Typography
+                      gutterBottom
+                      variant="h6"
+                      align="center"
+                      component="div"
+                    >
+                      Open Courses
+                    </Typography>
+                    <Typography align="center">
+                      Courses on Android, Game dev projects and more
+                    </Typography>
+                  </CardContent>
+                </Card>
+              </Grid>
+            </Grid>
+          </Container>
+
+          {/* ********************** Section 4 ******************* */}
+          <Container sx={{ mt: 10 }} maxWidth="sm">
+            <Typography
+              variant="h5"
+              component="h6"
+              align="center"
+              color="textPrimary"
+              gutterBottom
+            >
+              How can Meraki help with your concerns?
+            </Typography>
+          </Container>
+
+          <Container className={classes.cardGrid} maxWidth="lg">
+            <Grid container spacing={4}>
+              <Grid item xs={12} ms={6} md={4}>
+                <Card className={classes.card}>
+                  <CardMedia
+                    className={classes.cardMedia}
+                    component="img"
+                    src={require("./assets/learn-python.svg")}
+                    alt="green iguana"
+                  />
+                  <CardContent className={classes.cardConten}>
```

Comment:
Did you mean to use the `cardContent` class here? (There are a number of occurrences.)
```suggestion
                  <CardContent className={classes.cardContent}>
```